### ISSUE TITLE
npm WARN deprecated @discordjs/collection@0.5.0: no longer supported

### ISSUE THREAD

**Issue by Vinnybrunn00** [open-wa/wa-automate-nodejs#3115]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Multi-device and I have set multiDevice to true in my config OR am using the --multi-device flag

### What type of host account are you experiencing this issue on?

Personal account (normal)

### Mode

EASY API/CLI

### Steps To Reproduce

I created a folder called `bots` for my projects, when I include the code and run the command `npm i --save @open-wa/wa-automate@latest` I get the above error.

### DEBUG INFO

```markdown
› npm i --save @open-wa/wa-automate@latest 
npm WARN deprecated @discordjs/collection@0.5.0: no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm ERR! code 1
npm ERR! path /home/vinny/bott/node_modules/puppeteer
npm ERR! command failed
npm ERR! command sh -c node install.js

npm ERR! A complete log of this run can be found in: /home/vinny/.npm/_logs/2023-05-10T23_25_28_928Z-debug-0.log
```

### Environment

```markdown
- OS: Ubuntu 22.04.2
- Node: v18.16.0
- npm: 9.6.6
```

### Screenshots/Logs

![Captura de tela de 2023-05-10 20-30-38](https://github.com/open-wa/wa-automate-nodejs/assets/91799009/b3bdd4ab-6b29-4d43-861b-4954fc2315df)

**Comment by smashah** [comment]
the issue you are seeing doesnt really have anything to do with those dependency warnings. Please open and share the debug log

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.66.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.65.1...4.66.0) (2023-05-11)

- Release 4.66.0 [`9c28b84`](https://github.com/open-wa/wa-automate-nodejs/commit/9c28b8416e36c9eeba86ad65a5108b4e2963a592)
- ⬆️  upgrade uuid dep #3115 [`f9ef8bd`](https://github.com/open-wa/wa-automate-nodejs/commit/f9ef8bd6239b979c26a35922e22065ce1f08b467)
- ⬆️ upgrade discord collection dep #3115 [`ec6561a`](https://github.com/open-wa/wa-automate-nodejs/commit/ec6561a60d748a84e4ffb88a4cb7c2458569e3d6)
- 🗑️ depreacted lists and buttons till further notice #3116 [`45594e0`](https://github.com/open-wa/wa-automate-nodejs/commit/45594e0f1a40e2a70335a6ac659e80c4bdc80681)
- updated types-only package [`f7d172d`](https://github.com/open-wa/wa-automate-nodejs/commit/f7d172deade94cae3551cf461ab3b627e37f9479)
- 💚 fix build [`fa54448`](https://github.com/open-wa/wa-automate-nodejs/commit/fa54448e608f509cc5b5e7edd9a75aab02a9c011)
- 🥚 patches update [`ef6779b`](https://github.com/open-wa/wa-automate-nodejs/commit/ef6779b032b2dbb2f3075f3a68631eb209881ca5)
- updated types-only package [`d79d35d`](https://github.com/open-wa/wa-automate-nodejs/commit/d79d35d55936a535ebe9f94533eef073f8bf8ba7)

**Comment by Vinnybrunn00** [comment]
Hellow @smashah , i solved the problem. I had not installed the dependencies as `root`.

`uuid` was already updated.

thanks for helping :)
